tests/accessor/accessor_common.h

```diff
@@ -139,30 +139,70 @@ inline auto add_vectors_to_type_pack(StrNameType type_name) {
                                   "vec<" + type_name + ", 16>");
 }
 
+template <accessor_type AccT>
+struct tag_factory {};
```

Comment:
```suggestion
template <accessor_type AccType>
struct tag_factory {
  static_assert(AccType != AccType, "There is no tag support for such accessor type");
};
```

1. We can avoid `tag_factory<accessor_type::local_accessor>` in such way at the cost of entire struct instantiation support
2. Let's use `AccType` for naming consistency through the header

Reply:
Good idea. Applied it.